clients/acm.js

```diff
@@ -0,0 +1,14 @@
+require('../lib/node_loader');
+var AWS = require('../lib/core');
+var Service = require('../lib/service');
+var apiLoader = require('../lib/api_loader');
```

Comment:
Similarly here, `node_loader.js` and `browser_loader.js` already require `api_loader.js`, and `apiLoader` can already be accessed at `AWS.apiLoader`. `apiLoader` would also not need to be exported from `api_loader.js`.